appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt

```diff
@@ -114,12 +118,10 @@ internal sealed class ExperienceLifecycleEvent(
                 flatStepIndex?.let {
                     val step = experience.flatSteps[it]
                     hashMapOf(
-                        "interactionType" to interactionType.analyticsName(),
-                        "interactionData" to hashMapOf(
-                            "formResponse" to step.formState.formItems.map { formItem ->
-                                formItem.analyticsProperties()
-                            }
-                        ),
+                        INTERACTION_TYPE_KEY to interactionType.analyticsName(),
+                        INTERACTION_DATA_KEY to when (interactionType) {
+                            FORM_SUBMITTED -> step.formState
```

Comment:
this is where we now just store the form state object directly in the properties and have a custom serializer adapter in Moshi that controls how the output looks (see new `ExperienceStepFormStateAdapater`)